typescript-deno-plugin/src/plugin.ts

```diff
@@ -182,6 +186,22 @@ export class DenoPlugin implements typescript.server.PluginModule {
       }
 
       // Get typescript declaration File
+      if (
```

Comment:
This seems to only run when the project is initialized.

So if the configuration changes, it doesn’t seem to be notified.

I think it might be better to change env in here

https://github.com/denoland/vscode_deno/blob/65a0928ef9a5db1640ce2bd95f899a8c72ba2ff7/typescript-deno-plugin/src/plugin.ts#L557-L559

https://github.com/denoland/vscode_deno/blob/65a0928ef9a5db1640ce2bd95f899a8c72ba2ff7/typescript-deno-plugin/src/plugin.ts#L546-L552